client/verta/verta/client.py

```diff
@@ -890,6 +890,7 @@ def get_or_create_endpoint(self, path=None, description=None, workspace=None, id
         workspace = self._set_from_config_if_none(workspace, "workspace")
         if workspace is None:
             workspace = self._get_personal_workspace()
+        Endpoint._get_or_create_by_name(self._conn, path, lambda name: None, lambda name: Endpoint._create( self._conn, self._conf, workspace, path, description))
```

Comment:
I am assuming the `lambda name: None` is replace in https://github.com/VertaAI/modeldb/pull/1113!